spafe/features/mfcc.py

```diff
@@ -6,39 +6,53 @@
   For a copy, see <https://github.com/SuperKogito/spafe/blob/master/LICENSE>.
 
 """
+from typing import Optional
+
 import numpy as np
 from scipy.fftpack import dct
-from ..utils.cepstral import normalize_ceps, lifter_ceps
-from ..utils.exceptions import ParameterError, ErrorMsgs
-from ..fbanks.mel_fbanks import inverse_mel_filter_banks, mel_filter_banks
-from ..utils.preprocessing import pre_emphasis, framing, windowing, zero_handling
-
 
+from ..fbanks.mel_fbanks import inverse_mel_filter_banks, mel_filter_banks
+from ..utils.cepstral import normalize_ceps, lifter_ceps, NormalizationType
+from ..utils.converters import MelConversionApproach
+from ..utils.exceptions import ParameterError, ErrorMsgs
+from ..utils.filters import ScaleType
+from ..utils.preprocessing import (
+    pre_emphasis,
+    framing,
+    windowing,
+    zero_handling,
+    WindowType,
+)
+
+
+# TODO: the mel conversion approaches are only ["Oshaghnessy", "Lindsay"]
```

Comment:
Some conversion approaches were not very stable so I restricted the choices to the ones I deemed stable.

Reply:
Well then maybe you should remove references to the Beranek approach then? (If I understand everything correctly).

Reply:
I would rather not delete things and leave it to the user to decide :) so I will mention the Beranek appraoch too.